sources/packages/backend/workflow/test/2022-2023/fulltime-assessment/fulltime-assessment.e2e-spec.ts

```diff
@@ -97,9 +97,6 @@ describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}.`, () => {
     expect(
       calculatedAssessment.variables.calculatedDataTotalSecondResidence,
     ).toBe(expectedAssessmentData.secondResidenceCost);
-    expect(calculatedAssessment.variables.caclulatedDataTotalAssessedNeed).toBe(
-      expectedAssessmentData.totalAssessmentNeed,
```

Comment:
When identifying the root cause of why `totalAssessmentNeed` is not returned, I found that `caclulatedDataTotalAssessedNeed` is not present anymore in fulltime-assessment*.bpmn.

From the dev perspective, I believe we do not have to do anything here but we can share the usage of this property in various places. 

1. Dynamic Reports

![image](https://github.com/user-attachments/assets/e9b83625-6c8d-4eba-a783-092531285fb9)

2. IER 12
![image](https://github.com/user-attachments/assets/f6219ec5-1330-4cdd-8861-3005f981d31b)

3. It is used in the assessment data for FT. But not used in NOA. 

![image](https://github.com/user-attachments/assets/212620d5-d3d9-4e0f-a2f6-e21450331a7d)